explore/index.html

```diff
@@ -24,10 +24,10 @@ <h2 class="header-red">OME for Scientists</h2>
         <div class="row">
             <img class="medium-3 medium-offset-1 columns" alt="OMERO for Scientists" src="{{ site.baseurl }}/img/icons/user-groups_scientists.svg">
             <div class="medium-8 columns">
-                <p>Sign up for our <a href="http://qa.openmicroscopy.org.uk/registry/demo_account/">OMERO demo server</a> to test out the functionality with your own data. If happy, <a href="{{ site.baseurl }}/omero/institution/getting-started.html"> check how to start with OMERO in your institution.</a></p>
+                <p>Sign up for our <a href="https://demo.openmicroscopy.org/signup/">OMERO demo server</a> to test out the functionality with your own data. If happy, <a href="{{ site.baseurl }}/omero/institution/getting-started.html"> check how to start with OMERO in your institution.</a></p>
                 <p><a href="{{ site.baseurl }}/bio-formats/">Bio-Formats</a> lets you read image data from Java, <a href="https://imagej.net/Welcome" target="_blank">ImageJ</a>/<a href="https://fiji.sc/" target="_blank">Fiji</a>, GNU Octave/MATLAB, Python and R. If you're already using Fiji, it comes bundled with the <a href="https://docs.openmicroscopy.org/bio-formats/{{ site.bf.version }}/users/index.html#using-bio-formats-with-imagej-and-fiji">Bio-Formats plugin</a> automatically.</p>
                 <hr class="invisible">
-                <p><a class="button small" href="http://qa.openmicroscopy.org.uk/registry/demo_account/">Sign up for Demo Server</a>
+                <p><a class="button small" href="https://demo.openmicroscopy.org/signup/">Sign up for Demo Server</a>
```

Comment:
I am not sure about this. The omission of the http://qa.openmicroscopy.org.uk/registry/demo_account/ page will mean that the new signee is not going to read the Terms and Conditions, and have exagerrated expectations on the demo server.

Do we have to drop this http://qa.openmicroscopy.org.uk/registry/demo_account/ link because of the migration ?

Reply:
I don't know about the configurability of the `omero-signup` app but from my perspective, it would make sense to have these SLA terms migrated to https://demo.openmicroscopy.org/signup/ either inline or as a separate page.

That mirrors the traditional pattern where terms and conditions are usually available at sign up, usually with a checkbox indicating you have accepted them.